src/steps/format-json.js

```diff
@@ -0,0 +1,53 @@
+import dasherize from "dasherize";
+import camelize from "../util/camelize";
+import Resource from "../types/Resource";
+import Collection from "../types/Collection";
```

Comment:
The project's structure right now is that each step is a single function that returns a promise indicating when that step is done. Therefore, I don't think this code really fits as a step. Instead, it should probably be a file in `util`, something like `util/case-handling.js`. Then the camelize function can live there too.